### PR TITLE
enhanced logging of instance create to print wait time

### PR DESCRIPTION
**Pull request by manoj-ms**

### CODE REVIEW

**Comment by manoj-ms** [comment]
logging.debug('Waiting %d seconds for instance %s with ID %s to finish building', waitTime, instance_name, self.get_id()) prints how long it waits before timing out.

File: cloudpunch/ostlib/oscompute.py [hunk 1]
```diff
@@ -214,8 +214,9 @@ def create(self, instance_name, image_id, flavor_name, network_id,
                                                      security_groups=[secgroup_id])
         logging.debug('Created instance %s with ID %s using image %s and flavor %s',
                       instance_name, self.get_id(), image_id, flavor_name)
-        logging.debug('Waiting for instance %s with ID %s to finish building',
-                      instance_name, self.get_id())
+        waitTime=2*retry_count
```
Comment (JacobCalmes) [inline cloudpunch/ostlib/oscompute.py]: flake8 requires white space around the =

**Review by JacobCalmes — CHANGES_REQUESTED** [review]

File: cloudpunch/ostlib/oscompute.py (hunk 1 above)
Comment (manoj-ms) [reply cloudpunch/ostlib/oscompute.py]: Code is now flake8 compliant.

**Comment by manoj-ms** [comment]
requested change has been incorporated.

**Comment by manoj-ms** [comment]
Changes implemented!

Sent from my iPhone

On 29-Mar-2018, at 9:14 PM, Jacob Lube <notifications@github.com<mailto:notifications@github.com>> wrote:


@JacobLube requested changes on this pull request.

________________________________

In cloudpunch/ostlib/oscompute.py<https://github.com/target/cloudpunch/pull/22#discussion_r178098704>:

> @@ -214,8 +214,9 @@ def create(self, instance_name, image_id, flavor_name, network_id,
                                                      security_groups=[secgroup_id])
         logging.debug('Created instance %s with ID %s using image %s and flavor %s',
                       instance_name, self.get_id(), image_id, flavor_name)
-        logging.debug('Waiting for instance %s with ID %s to finish building',
-                      instance_name, self.get_id())
+        waitTime=2*retry_count


flake8 requires white space around the =

**Review by JacobCalmes — APPROVED** [review]

**Pull request merged**